client/tests/phpunit/Service/ParameterStoreServiceTest.php

```diff
@@ -37,4 +38,35 @@ public function getParameter()
 
         self::assertEquals('result', $sut->getParameter('test-flag'));
     }
+
+    /**
+     * @dataProvider parameterDataProvider
+     * @test
+     */
+    public function addParameter($parameterName, $parameterValue)
+    {
+        $ssmClient = self::prophesize(SsmClient::class);
+        $parameterPrefix = '/param-prefix/';
+        $ssmClient
+            ->putParameter(Argument::exact(
+                ['Name' => $parameterPrefix.$parameterName,
+                    'Value' => $parameterValue, ]))
+            ->shouldBeCalled()
+            ->willReturn([
+                'Tier' => 'Standard',
+                'Version' => 10,
+            ]);
+
+        $sut = new ParameterStoreService($ssmClient->reveal(), $parameterPrefix, '/flag-prefix/');
+
+        $sut->putParameter($parameterName, $parameterValue);
+    }
+
+    public function parameterDataProvider()
+    {
+        return [
+            'document sync set to true' => ['document-sync', 1],
+            'checklist sync set to false' => ['checklist-sync', 0],
+        ];
+    }
```

Comment:
And for completeness sake we should cover the `putFlag` function we wote under a test here (basically the same layout as this but varying the `$flagPrefix` var rather than the `$parameterPrefix` var.

We should also add this test to the API app seeing as we duplicated this code.